facebook_scraper/utils.py

```diff
@@ -43,3 +46,36 @@ def make_html_element(html: str, url=DEFAULT_URL) -> Element:
 
 def html2text(html: str) -> str:
     return _html2text(html)
+
+
+month = r"Jan(?:uary)?|" \
+        r"Feb(?:ruary)?|" \
+        r"Mar(?:ch)?|" \
+        r"Apr(?:il)?|" \
+        r"May|" \
+        r"Jun(?:e)?|" \
+        r"Jul(?:y)?|" \
+        r"Aug(?:ust)?|" \
+        r"Sep(?:tember)?|" \
+        r"Oct(?:ober)?|" \
+        r"Nov(?:ember)?|" \
+        r"Dec(?:ember)?|" \
+        r"Yesterday|" \
+        r"Today"
+date = f"({month}) " + r"\d{1,2}"
+hour = r"\d{1,2}"
+minute = r"\d{2}"
+period = r"AM|PM"
+exact_time = f"({date}) at {hour}:{minute} ({period})"
+relative_time = r"\d{1,2} \w+"
```

Comment:
This seems a bit vague since it is being matched with the text from the whole post and not just a date.
What exactly is this trying to match?

Reply:
According to a previous comment it could be 16 hrs or 16h so maybe:
```suggestion
relative_time = r"\b\d{1,2}(?:h| hrs)"
```

Reply:
The only date I found is in the entire post text, and I only saw dates that have explicit month and day of month specification and a time, days that have 'Today' or 'Yesterday' and a time, and just relative time dates (16 hours ago, hours is the only case I found)